lua/nvim-treesitter/textobjects/select.lua

```diff
@@ -1,15 +1,15 @@
 local api = vim.api
-local configs = require'nvim-treesitter.configs'
-local parsers = require'nvim-treesitter.parsers'
-local queries = require'nvim-treesitter.query'
+local configs = require "nvim-treesitter.configs"
```

Comment:
This change caused some lines in this file to get longer so it was time for a auto-format.